src/components/networked.js

```diff
@@ -199,6 +199,15 @@ AFRAME.registerComponent('networked', {
     NAF.entities.registerEntity(networkId, this.el);
   },
 
+  applyPersistentFirstSync: function() {
```

Comment:
is this not being called yet? it feels like if the intent is for the application to call this, theoretically we should be able to encapsulate this by dealing with it in `init` on `networked`

Reply:
For reference, responded in other PR: https://github.com/mozilla/hubs/pull/1556#discussion_r306551654